messages/en.json

```diff
@@ -7,19 +7,19 @@
   "store/product-list.message.noLongerAvailable": "This product is no longer available.",
   "store/product-list.message.remove": "remove",
   "store/product-list.quantity-selector.remove": "Remove",
-  "store/product-list.quantity-label": "{quantity} un.",
+  "store/product-list.quantity-label": "{quantity} unit(s)",
   "store/product-list.delete-button.default-label": "Remove Product",
   "store/product-list.priceChanged": "Changed",
   "store/product-list.priceOptions": "Price options",
-  "store/product-list.changePrice": "Change price",
-  "store/product-list.revertToOriginal": "Revert to original",
+  "store/product-list.changePrice": "Change Price",
+  "store/product-list.revertToOriginal": "Revert to Original",
   "store/product-list.originalPrice": "Original price",
   "store/product-list.changeTo": "Change to",
-  "store/product-list.manualPricePlaceholder": "Type a monetary value",
-  "store/product-list.applyManualPrice": "ok",
+  "store/product-list.manualPricePlaceholder": "Enter an amount",
+  "store/product-list.applyManualPrice": "OK",
   "store/product-list.quantitySelector.label": "{quantity} {measurementUnit}",
   "store/product-list.quantitySelector.maxValueLabel": "{quantity} {measurementUnit} +",
-  "store/product-list.quantityUnitMultiplierMismatch": "This product is sold by fractions of {unitMultiplier}{measurementUnit}. Given that, the inserted quantity has been rounded to {roundedValue}{measurementUnit}.",
+  "store/product-list.quantityUnitMultiplierMismatch": "This product is sold by fractions of {unitMultiplier}{measurementUnit}. That is why the entered quantity has been rounded to {roundedValue}{measurementUnit}.",
```

Comment:
This text is being used in our tests, changing this label breaks some tests, suggest keeping the same translation or/and fixing the text on tests

Reply:
https://github.com/vtex-apps/product-list/blob/8bb6492114ed0b78ea17da36d86c51e84600cb04/react/components/__tests__/QuantityStepper.test.tsx#L43

Reply:
Done, @iago1501 !